httpd/filters.go

```diff
@@ -0,0 +1,147 @@
+package httpd
```

Comment:
I think this falls into "bike shedding" territory for me.  Personally, I'm a fan of files that are named after their responsibility.  It allows me to quickly navigate a project.  I find that right now the only way to navigate our project is to Ack for everything, which for me frustrates me (again, this is a personal preference for me).  I know I'm a minority in the go community when it comes to "everything goes in one big file".

If i'm the minority here I'm happy to merge the files.  I would love to have a conversation over lunch one day about it though and explain my point of view and my experience with large go projects with more files than less.  <Insert bike shedding meme here> :-).